Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Tests/FinalizeGltfContainerLoadingSystemShould.cs

```diff
@@ -122,10 +128,11 @@ public async Task InstantiateVisibleMeshesColliders()
             GltfContainerAsset promiseAsset = component.Promise.Result.Value.Asset;
 
             Assert.That(promiseAsset.VisibleMeshesColliders.Count, Is.EqualTo(196));
-            Assert.That(promiseAsset.VisibleMeshesColliders.All(c => c.gameObject.layer == PhysicsLayers.ON_POINTER_EVENT_LAYER), Is.True);
+            Assert.That(promiseAsset.VisibleMeshesColliders.All(c => c.Collider.gameObject.layer == PhysicsLayers.ON_POINTER_EVENT_LAYER), Is.True);
         }
 
         [Test]
+        [RequiresPlayMode]
```

Comment:
Using `RequiresPlayMode` makes most of edit mode tests skip in the CI. I could never figure out the origin of this, it may even be a Unity bug.

As a quick fix, I suggest moving these tests to the PlayMode assembly

Reply:
I changed the code a bit to get rid of necessity for Play Mode Tests